src/status_im/network/core.cljs

```diff
@@ -82,25 +94,29 @@
      (handler data cofx))))
 
 (fx/defn save
-  [{{:network/keys [manage] :account/keys [account] :as db} :db
+  [{{:networks/keys [manage] :account/keys [account] :as db} :db
     random-id-generator :random-id-generator :as cofx}
-   {:keys [data success-event on-success on-failure]}]
+   {:keys [data success-event on-success on-failure network-id chain-id-unique?]}]
   (let [data (or data manage)]
     (if (valid-manage? data)
-      (let [{:keys [name url chain network-id]} data
-            network      (new-network (random-id-generator)
+      (let [{:keys [name url chain] chain-id :network-id} data ;rename network-id from UI to chain-id
```

Comment:
please, move comments to the line above commented code and in general keep line less than or equal 80 chars